src/TimeColumn.js

```diff
@@ -129,7 +130,25 @@ export default class TimeColumn extends Component {
     // excluding the time gutter TimeColumn from having a time indicator.
     if (!dragThroughEvents) return;
 
-    const now = dates.now(nowTimezone);
+    let now = dates.now(nowTimezone);
+
+    const dayStart = dates.startOf(now, 'day');
+    const dayEnd = dates.endOf(now, 'day');
+
+    const dayStartTimezoneOffset = dayStart.getTimezoneOffset();
+    const dayEndTimezoneOffset = dayEnd.getTimezoneOffset();
+    const nowTimezoneOffset = now.getTimezoneOffset();
+
+    // This is generally at 2:00am on the day we fall back to standard time
+    const isAfterFallBackTimeShift =
```

Comment:
Isn't the first part of this condition the same thing as `isFallingBack`? Can we do this instead?
```ts
const isAfterFallBackTimeShift = isFallingBack && nowTimezoneOffset === dayEndTimezoneOffset;
```
I suppose I'm not sure we even need to care about `dayStartTimezoneOffset`.

Reply:
Then the condition on line 149 would just be `if (isAfterFallBackTimeShift)`

Reply:
Yeah, the current solution is more specific than it needs to be. I'll clean it up.